cylc/flow/tui/app.py

```diff
@@ -435,7 +436,8 @@ def update_header(self):
             ('Y', 'y'),
             ('G', 'l'),
             ('B', 'c'),
-            ('title', ' Tui')
+            ('title', ' Tui '),
+            ('diminished', CYLC_VERSION),
```

Comment:
[minor unrelated change] Add the cylc version into the Tui header - https://github.com/cylc/cylc-flow/pull/6671/commits/f3ee55c5319b127abd98554c4ef328106e501402